libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -624,6 +624,8 @@ const AP_Scheduler::Task AP_Vehicle::scheduler_tasks[] = {
 #endif
     SCHED_TASK(send_watchdog_reset_statustext,         0.1,     20, 225),
 #if HAL_WITH_ESC_TELEM
+    // This update function is responsible for checking timeouts and invalidating the ESC telemetry data.
+    // Be mindful of this if you are planning to reduce the frequency from 100Hz.
```

Comment:
On plane/rover this will run at 50Hz, I assume that is still OK?

Reply:
Yeah, that's perfectly okay. The timeout is configured to be 1s, and has a worst case actual time of 1s + 1/freq. Even as low as 10Hz should be perfectly fine IMO, especially on platforms with low main-loop rates.

Reply:
Thanks